format/src/main/scala/org/scalafmt/internal/SyntaxTokens.scala

```diff
@@ -261,6 +261,27 @@ object SyntaxTokens {
     def tokensLeftParen: LeftParen = tree.find[LeftParen].get
     def tokensRightParen: RightParen = tree.findAfter[RightParen](_.expr).get
   }
+
+  // == Type ==
+  implicit class XtensionTypeByNameSyntax(val tree: Type.ByName)
```

Comment:
Just noticed these fields are public which means `Type.ByName.tree` is valid, let's make them `private`